src/main/java/com/jupiter/boot_csv_parser/Parser.java

```diff
@@ -0,0 +1,51 @@
+package com.jupiter.boot_csv_parser;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import com.jupiter.boot_csv_parser.model.Record;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Parser {
+
+    public List<Record> parse() {
+        try {
+            String path = getClass().getClassLoader().getResource("Reviews.csv").getFile();
+            BufferedReader reader = new BufferedReader(new FileReader(path));
+            long timeBefore = System.currentTimeMillis();
+            List<Record> records = new ArrayList<>();
+            reader.readLine();
+            while(reader.ready()) {
+                Record record = convertToRecord(reader.readLine().split(","));
+                records.add(record);
+            }
+            System.out.println(System.currentTimeMillis() - timeBefore);
+            reader.close();
+            return records;
+        } catch (IOException e) {
+            throw new RuntimeException();
+        }
+    }
+
+    private Record convertToRecord(String[] arr) {
+        Record record = new Record();
+        record.setId(Long.valueOf(arr[0]));
+        record.setProductId(arr[1]);
+        record.setUserId(arr[2]);
+        record.setProfileName(arr[3]);
+        record.setHelpfulnessNumerator(arr[4]);
```

Comment:
do you need all these fields? can we use only specific fields for this task?

Reply:
done